01-getting-started/end-to-end-example.ipynb

```diff
@@ -35,7 +35,7 @@
    "metadata": {},
```

Comment:
<p>why is this formatted differently?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Ishan, liked this formatting, and send it this way, its the same info</p>